Mayhemfile

```diff
@@ -0,0 +1,32 @@
+project: rust-lexical
+target: rust-lexical
+
+cmds:
```

Comment:
Mayhemfiles should take only one command for a target. You should either
a. Remove the rest of the commands, or
b. Split the Mayhemfile into several Mayhemfiles, one for each target binary.
Then, make sure that the `target` field reflects the binary being run (if your binary is called `parse-integer-i8`, then the target should also be called `parse-integer-i8`.

Reply:
For more details on the second option, see here: https://community.forallsecure.com/t/error-the-mayhem-for-code-scan-detected-the-mayhem-run-for-your-target-was-unsuccessful-and-multiple-docker-files-in-the-repository/350/4

Reply:
I will check this out. I would like to fuzz the multiple targets in a seamless way. However, if I have to split the mayhemfile into 24 files, I believe that will be messy.